### PR TITLE
Feature/remove needs counter culture

### PR DESCRIPTION
**Pull request by go8soft**
Task: https://trello.com/c/dJS5FsI0/109-remove-cached-task-counts-from-contacts-model

### CODE REVIEW

File: app/models/contact.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 class Contact < ApplicationRecord
   include PgSearch::Model
+  self.ignored_columns = ['needs_count', 'uncompleted_needs_count', 'completed_needs_count']
```
Comment (timwis) [inline app/models/contact.rb:5]: Why do we need this if we've removed the columns?

File: db/seeds.rb [hunk 2]
```diff
@@ -63,7 +63,7 @@
                             contact: contact,
                             user: [user, nil].sample,
                             is_urgent: [true, false].sample,
-                            completed_on: [nil, [1,2,3].sample.days.ago].sample
+                            status: ['to_do', 'complete'].sample
```
Comment (timwis) [inline db/seeds.rb:66]: Shouldn't we use `Need.status.sample` instead so other statuses get included in the seeds?

File: app/models/contact.rb (hunk 1 above)
Comment (go8soft) [reply app/models/contact.rb:5]: Not necessarily, just good practice. Get it out of here?

Comment (go8soft) [reply app/models/contact.rb]: Done.

File: db/seeds.rb (hunk 2 above)
Comment (go8soft) [reply db/seeds.rb]: Thanks

**Comment by go8soft** [comment]
> Looks good, just left a couple comments that we might want to fix before merging
Thanks, I applied the changes.

**Pull request merged**